jobs/myOpenBugs.js

```diff
@@ -1,40 +1,117 @@
-'use strict';
+'use strict'
+
 const utils = require('../utils');
 
 module.exports = {
   name: 'myOpenBugs',
-  description: 'lists number of open bugs that a user has',
+  description: 'lists number of open bugs of a given priority that the entire project, a user or a team has',
   type: 'responsive',
   dependencies: [
     'jira'
   ],
   userInfoNeeded: [
-    'jiraUserId'
+    'jiraUserId',
+    'jiraTeam'
   ],
   hiddenFromHelp: false,
   phrases: [
-    'open bugs',
-    'how many open bugs do i have',
-    'open bug count'
+    'how many p2s are there',
+    'what are my bugs',
+    'what are my p2 bugs',
+    'what are my p3 bugs',
+    'what are my team\'s p2s',
+    'give me the p2 p3 count',
+    'what is the total p2 p3 count',
+    'total bug count',
+    'bug count',
+    'p2 p3 count',
+    'p2 count',
```

Comment:
the fuzzy matcher is pretty good so you don't need so many similar phrases. It doesn't really hurt, but it's not necessary.